diff -u -r esc/src/lib/coolkey/CoolKey_Message.cpp esc/src/lib/coolkey/CoolKey_Message.cpp
--- esc/src/lib/coolkey/CoolKey_Message.cpp	2018-05-04 05:03:18.000000000 +0800
+++ esc/src/lib/coolkey/CoolKey_Message.cpp	2023-04-28 21:48:22.000000000 +0800
@@ -213,7 +213,7 @@
     if(!aKey.length())
         return;
 
-    if(aSize <=0 || !aValue)
+    if(*aSize <=0 || !aValue)
         return;
 
     string data = "";
diff -u -r esc/src/lib/nss-http-client/request.cpp esc/src/lib/nss-http-client/request.cpp
--- esc/src/lib/nss-http-client/request.cpp	2018-05-05 01:20:00.000000000 +0800
+++ esc/src/lib/nss-http-client/request.cpp	2023-04-28 21:46:23.000000000 +0800
@@ -192,7 +192,7 @@
     }
 
     _bodyLength = size;
-     sprintf(_body,body);
+		sprintf(_body, "%s", body);
     //_body = (char *)body;
 
     return PR_TRUE;
